README.md

```diff
@@ -41,11 +42,11 @@ To create a migration, execute `db-migrate create` with a title. `node-db-migrat
 
 ```javascript
 exports.up = function (db, callback) {
-  callback();
```

Comment:
Yea i added it back b/c i thought we had agreed that it should be added to the generated code and leave the docs as they were. Basically if it generates `callback();`, the docs should reflect that. Now they do. Previously the docs showed that they generated `callback();`, but the migrator didn't actually generate that callback.

The spacing is off b/c I added it before I had switched from tabs to spaces.